Microsoft.Research/ClousotMain/ClousotMain.cs

```diff
@@ -795,6 +796,10 @@ private bool FilterPreconditionsAccordingToWarningLevel(BoxedExpression exp)
 
       public int Analyze()
       {
+        // String formatting in the analyzer does not specify a culture, therefore some tests fail on systems with non-English culture with e.g. "expected: 2.0, actual: 2,0"
+        // Setting the threads culture should be removed as soon as issue #149 is fixed.
+        Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
```

Comment:
The other way is to fix thousands of formatting statements where the culture is missing. 

Pros for this way of fixing (until all the code is culture-specific):
- All the code has never been verified using another culture than US(~Invariant). 
- The only thing affected by this would be the UI - but the analyzer does not have one.